homeassistant/components/homekit_controller/climate.py

```diff
@@ -442,6 +479,12 @@ def supported_features(self):
         if self.service.has(CharacteristicsTypes.TEMPERATURE_TARGET):
             features |= SUPPORT_TARGET_TEMPERATURE
 
+        if self.service.has(CharacteristicsTypes.TEMPERATURE_COOLING_THRESHOLD):
```

Comment:
I wonder if we should make this check an AND. We only support range where both threshold charactersitics are available AND auto mode is available. I hope in the wild there aren't devices that don't have 1 of these but not the other (they wouldn't make sense). But i'd still prefer to make the check an AND just in case, don't you think?

Reply:
I agree.  I had grabbed the code from the Heater/Cooler section and meant to go back and change that.